### DISCUSSION TITLE
Simplify directus_users table

### DISCUSSION THREAD

**Discussion by cdwmhcc** [directus/directus#20247] [Feature Requests]
### Summary

After updating to the new version, I found that the users table has more fields such as `theme_light` `theme_dark` `theme_light_overrides` `theme_dark_overrides`. It is a good thing for the user experience that directus apps provide css variable configuration for changing the interface color, But these fields exist entirely to adapt to the app. I think there is a problem with the design idea.The system should be decoupled modules rather than coupled modules. Just like in the future, Directus will try its best to extract modules into the application market instead of integrating them all into the core.

When the user needs first_name, the user can create a string field in the users table, and the same is true for theme_light. 

There are many preset fields in the users table, and I think removing these fields will not have any impact.
`theme_light`
`theme_dark`
`theme_light_overrides`
`theme_dark_overrides`
`title`
`description`
`location`
`tags`
``
Even `first_name` and `last_name` are not required.

### Motivation

Make some subtractions on the core layer of the system for the upcoming expansion market

### Detailed Design

1. The optimal solution for the core of the system is to remove non-essential fields in the users table.
2. Considering the large number of existing directus users, providing administrators with manual removal of fields is the lowest cost solution.

### Requirements List

Remove them or have a removal function.

### Drawbacks

I don't think there would be any ill effects if the admin was only given the option to remove certain fields from user.

### Alternatives

no

### Adoption Strategy

I don't think there would be any ill effects if the admin was only given the option to remove certain fields from user.

**Comment by Seinnd** [comment]
It's quite possible that I understand the request completely wrong, but why exactly is it not enough for you to be able to set corresponding read permission at role level on a field basis for the system collection **directus_user**?
![image](https://github.com/directus/directus/assets/22862290/cb4d2a55-4e84-4cd8-bde8-7979c14db787)

**Reply by vitalijalbu** [reply]
I also have this issue, i set **ONLY** some fields, ant then on API response i see all of them, **password included**

**Reply by br41nslug** [reply]
> I also have this issue, i set ONLY some fields, ant then on API response i see all of them, password included

@vitalijalbu This is a feature request not an issue, from what you're describing please do make an issue for it 😄

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
